{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Product defects","children":[{"name":"Timed out retrying after 10000ms: Expected to find element: `h1`, but never found it.","children":[{"name":"Check App1 and App2 elements","uid":"8758e4f92a3f73d1","parentUid":"23fd062f6b4cc0af7a7ed54a88a82e22","status":"failed","time":{"start":1698478876027,"stop":1698478886498,"duration":10471},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"23fd062f6b4cc0af7a7ed54a88a82e22"},{"name":"The following error originated from your application code, not from Cypress.\n\n  > Additional keys were returned from `getServerSideProps`. Properties intended for your component must be nested under the `props` key, e.g.:\n\n\treturn { props: { title: 'My Title', content: '...' } }\n\nKeys that need to be moved: userId, id, title, completed.\nRead more: https://nextjs.org/docs/messages/invalid-getserversideprops-value\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check that Shop navigation link works","uid":"bb9ec68810ac367e","parentUid":"9923ecdd3da820d4b0811512ad94f2f1","status":"failed","time":{"start":1698481560378,"stop":1698481564485,"duration":4107},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"Check that Shop text navigation link works","uid":"1bb09d8f65f910c7","parentUid":"9923ecdd3da820d4b0811512ad94f2f1","status":"failed","time":{"start":1698481595038,"stop":1698481599145,"duration":4107},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":3,"retriesStatusChange":true,"parameters":[],"tags":[]}],"uid":"9923ecdd3da820d4b0811512ad94f2f1"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"},{"name":"Test defects","children":[{"name":"The following error originated from your application code, not from Cypress.\n\n  > Cannot find module 'app1/Button'\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check elements exist","uid":"a414f70f6aa3ac69","parentUid":"d2140daa7579ccd0f3e01294a2cdf6c2","status":"broken","time":{"start":1698481157531,"stop":1698481158055,"duration":524},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":true,"retriesCount":2,"retriesStatusChange":true,"parameters":[],"tags":[]}],"uid":"d2140daa7579ccd0f3e01294a2cdf6c2"},{"name":"`cy.visit()` failed trying to load:\n\nhttp://localhost:3002//shop\n\nThe response we received from your web server was:\n\n  > 500: Internal Server Error\n\nThis was considered a failure because the status code was not `2xx`.\n\nThis http request was redirected '1' time to:\n\n  - 308: http://localhost:3002/shop\n\nIf you do not want status codes to cause failures pass the option: `failOnStatusCode: false`","children":[{"name":"Check the header content of Shop page","uid":"1387b9d4221555ad","parentUid":"fc7d2bf7fe16529721b0bcd46f598781","status":"broken","time":{"start":1698481570594,"stop":1698481571256,"duration":662},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":true,"retriesCount":2,"retriesStatusChange":true,"parameters":[],"tags":[]}],"uid":"fc7d2bf7fe16529721b0bcd46f598781"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}